app/services/feed_query_service.rb

```diff
@@ -18,12 +25,15 @@ def list
     list = list.mark(mark) if mark
     list = list.sfw if sfw_filter?
     list = list.blocking(blocked)
-    list = list.select(kind_filter[:ratio], &kind_filter[:proc]) if kind_filter
```

Comment:
Probably want the split feeds to be behind a flag, so we can do dual-writes until the data is imported and then flip the switch